cloud/docker_client.go

```diff
@@ -107,6 +107,7 @@ func (c *dockerClientImpl) generateClient(h *host.Host) (*docker.Client, error)
 func (c *dockerClientImpl) changeTimeout(h *host.Host, newTimeout time.Duration) (*docker.Client, error) {
 	var err error
 	c.httpClient.Timeout = newTimeout
+	c.client = nil // don't want to use cached client
```

Comment:
Why not?

Reply:
This seemed like an error to me--the point of changeTimeout is to create a new httpClient, and presumably a new docker client (c.client) with this updated httpClient. If c.client was previously set, then in generateClient we won't create a new docker one with the updated httpClient. If we don't need the docker client to have an updated httpClient then we don't need this change.